hack/tools/release/notes.go

```diff
@@ -90,8 +93,15 @@ func firstCommit() string {
 }
 
 func run() int {
-	lastTag := lastTag()
-	cmd := exec.Command("git", "rev-list", lastTag+"..HEAD", "--merges", "--pretty=format:%B") //nolint:gosec
+	var lastTag string
+	var cmd *exec.Cmd
+	if *since != "" && *until != "" {
+		lastTag = fmt.Sprintf("%s - %s", *since, *until)
+		cmd = exec.Command("git", "rev-list", "HEAD", "--since=\""+*since+"\"", "--until=\""+*until+"\"", "--merges", "--pretty=format:%B") //nolint:gosec
+	} else {
+		lastTag := getLastTag()
```

Comment:
:= will lead to a new variable and won't overwrite lastTag in the function scope